python/lsst/meas/extensions/psfex/psfex.py

```diff
@@ -651,7 +283,7 @@ def showPsf(psf, set, ext=None, wcsData=None, trim=0, nspot=5,
         mosaic.writeFits(outFile)
 
 
-def getLsstFlags(tab=None):
+def getFlags(tab=None):
     flagKeys = [
         "base_PixelFlags_flag_edge",
         # "base_PixelFlags_flag_interpolated",
```

Comment:
Do we want to keep these commented out flags?

Reply:
Hmm... I think I'll leave them for future reference. There's no useful info in the git blame history, so I don't know why they were commented out.